PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -1,5 +1,7 @@
 package com.microsoft.azuretools.azureexplorer.forms;
 
+import com.microsoft.azuretools.telemetry.AppInsightsClient;
+import com.microsoft.intellij.hdinsight.messages.HDInsightBundle;
```

Comment:
They are used differently. Will refactor it later.

---
In reply to: [120865283](https://github.com/Microsoft/azure-tools-for-java/pull/603#discussion_r120865283) [](ancestors = 120865283)